Writerside/topics/Sessions.md

```diff
@@ -0,0 +1,129 @@
+# Sessions
+
+Sessions are typically used to represent users after a successful login.
+
+> A session produced by an authentication service typically represents a user but is only populated with account
```

Comment:
This is a good place to link out to a document that describes this in more detail. I don't think that document exists yet, but maybe we can add a todo?